vignettes/vrd_vignette.Rmd

```diff
@@ -0,0 +1,259 @@
+---
+title: "vrd_vignette"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{vrd_vignette}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+# Pulling Data from the API
+
+This package allows access to the Canada Governments Vehicle Recall Database.  It provides information of items that have been recalled that are or pertain to vehicles (such as tires, or child carriers).  The API provides information on the manufacturer, make, model, year manufactured, recall date, number of recalls, as well as information on why the recall happened.
```

Comment:
This package is an API wrapper for the Government of Canada Vehicle Recalls Database. The Vehicles Recall Database is used by the Defect Investigations and Recalls Division to record and publish recalls of vehicles, tires, and child car seats.

Pulled the last sentence off of the description page for the API.